sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -98,20 +98,24 @@ internal class NextLinkOperationImplementation : IOperation
             AssertNotNull(pipeline, nameof(pipeline));
 
             // TODO: Once we remove NextLinkOperationImplementation from internal shared and make it internal to Azure.Core only, we can access the internal members from RehydrationToken directly
```

Comment:
Please open an issue to track and reference it here.   I think removing serialization here is an important goal and I don't want to lose track.

Reply:
Updated https://github.com/Azure/azure-sdk-for-net/issues/43260, and added the link to comment